src/cli/cli_app.py

```diff
@@ -0,0 +1,29 @@
+from typing import Any, List, Mapping, Optional
+
+from src.cli.command import Command
+
+
+class CLIApp:
```

Comment:
I'm wondering why there are 2 classes instead of just one - ProtostarCLI. I wouldn't bother with dividing those if they aren't re-used anywhere

Reply:
It's easier to test `ReferenceDocsGenerator` and `ArgumentParserFacade`.